gson/src/main/java/com/google/gson/annotations/SerializedName.java

```diff
@@ -83,10 +83,12 @@
 public @interface SerializedName {
 
   /**
+   * A String that indicates the desired name of the field when it is serialized or deserialized.
```

Comment:
I'd go for brevity here too:

`The desired name of the field...`.

No need to say that it is a `String` since that is in the declared type.